### PR TITLE
registernach readme correction

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc reference : https://razorpay.com/docs/api/payments/recurring-payments/paper-nach/auto-debit/

### CODE REVIEW

File: documents/registernach.md [hunk 1]
```diff
@@ -130,7 +141,14 @@ $api->subscription->createSubscriptionRegistration(array('customer'=>array('name
 | amount*   | integer      | The amount to be captured (should be equal to the authorized amount, in paise) |
 | currency*   | string  | The currency of the payment (defaults to INR)  |
 | description*  | string      | A brief description of the payment.   |
-| subscription_registration   | array  | All keys listed [here](https://razorpay.com/docs/api/recurring-payments/paper-nach/auto-debit/#121-create-a-registration-link) are supported  |
+| subscription_registration["first_payment_amount"]      | integer  | The amount, in paise, the customer should be auto-charged in addition to the authorization amount. |
+| subscription_registration["auth_type"]           | string  | Possible values `nach`|
+| subscription_registration["max_amount"]           | integer  | The maximum amount, in paise, that a customer can be charged in one transaction.|
```
Comment (sofiya02) [inline documents/registernach.md:146]: -         All keys listed here are supported for customer

File: documents/registernach.md [hunk 2]
```diff
@@ -47,8 +47,19 @@ $api->order->create(array('amount' => 100, 'currency' => 'INR', 'method'=>'nach'
 | currency*   | string  | The currency of the payment (defaults to INR)  |
 | customerId*   | string      | The id of the customer to be fetched |
 | method*      | string  | Payment method used to make the registration transaction. Possible value is `nach`.  |
-| receipt      | string  | Your system order reference id.  |
-| token  | array  | All keys listed [here](https://razorpay.com/docs/api/recurring-payments/paper-nach/auto-debit/#112-create-an-order) are supported |
+| token["first_payment_amount"]  | integer  |  The amount, in paise, the customer should be auto-charged in addition to the authorization amount. |
```
Comment (sofiya02) [inline documents/registernach.md:50]: -         All keys listed here are supported for customer

**Review by sofiya02 — CHANGES_REQUESTED** [review]
Create order- Optional auth_type, bank_account, max_amount, expire_at, nach  are missing

File: documents/registernach.md (hunk 1 above)
Comment (ankitdas13) [reply documents/registernach.md]: added list link like before

File: documents/registernach.md (hunk 2 above)
Comment (ankitdas13) [reply documents/registernach.md]: added list link like before

**Review by sofiya02 — CHANGES_REQUESTED** [review]
Create order- Optional auth_type, bank_account, max_amount, expire_at, nach are missing

**Review by sofiya02 — APPROVED** [review]

**Review by sofiya02 — COMMENTED** [review]
- [Create an Order](https://razorpay.com/docs/api/payments/recurring-payments/paper-nach/auto-debit/#112-create-an-order) - Optional auth_type, bank_account, max_amount, expire_at, nach  are missing in readme

**Pull request merged**